tests/functional/monitoring/prometheus/alerts/test_ceph.py

```diff
@@ -87,6 +87,10 @@ def test_ceph_health(measure_stop_ceph_osd, measure_corrupt_pg, threading_lock):
     utilization.
     """
     api = prometheus.PrometheusAPI(threading_lock=threading_lock)
+    # the time to wait is increased because it takes more time for Ceph
+    # cluster to resolve its issues
+    health_wait = 420
```

Comment:
this value is set to 420, but these are minutes. 420/60 = 7, total 7 hours of wait .

```
 api.check_alert_cleared(
        label=target_label,
        measure_end_time=stop_time,
        time_min=health_wait,
    )
```

Reply:
sorry, ignore this.
time_min - it looks more like time in minutes than minimal time, which I expected to see as a min_time

As an enhancement, suggest to change this parameter name. Up to you, in this PR or in another.